tests/test_api.py

```diff
@@ -435,6 +436,57 @@ def test_c0_c1():
     assert not ke.compile("a[#c1]z").search("azure")
 
 
+def test_multi_range():
+    assert ke.re("[#-9..-1]") == "-[1-9]"
+    assert ke.re("[#-9..0]") == "-[1-9]|0"
+    assert ke.re("[#-9..-9]") == "-9"
+    assert ke.re("[#-99..-99]") == "-99"
+    assert ke.re("[#-9..-8]") ==  "-[8-9]"
+    assert ke.re("[#-9..9]") == "-[1-9]|\d"
+    assert ke.re("[#-0..-0]") == "0"
+    assert ke.re("[#-0..0]") == "0"
+    assert ke.re("[#0..-0]") == "0"
+    assert ke.re("[#-20..40]")
+    assert ke.re("[3 #-19..99 '.']") == "(?:(?:-(?:[1-9]|1\d)|(?:\d|[1-9]\d))\.){3}"
+
+    assert not ke.match("[#-9..-1]", "0")
+    assert not ke.match("[#-9..-1 #el]", "-10")
+    assert not ke.match("[#-9..-1 #el]", "-19")
+    assert not ke.match("[#-9..-1 #el]", "-99")
+    assert not ke.match("[#-9999..-100 #el]", "-99999")
+    assert not ke.match("[#-9999..-100 #el]", "-99")
+
+    assert ke.match("[#-9..-1]", "-9")
+    assert ke.match("[#-9..-1]", "-1")
+    assert ke.match("[#-9..-1]", "-5")
+    assert ke.match("[#-9999..-1000]", "-8000")
+    assert ke.match("[#-9999..-1000]", "-9999")
```

Comment:
these are all great, they test all the right positive cases, just make sure to test all similar negative cases as well (at least -9999-1, 1000+1)